src/components/gallery/Picture.jsx

```diff
@@ -1,8 +1,17 @@
-const Picture = () => {
+import Link from "next/link";
+import Image from "next/image";
+
+const Picture = ({ text, link, image }) => {
   return (
-    <div className="">
-      <p>Picture</p>
-    </div>
+    <Link href={link} target="_blank">
+      <div className="group relative flex justify-center items-center">
+        <p className="absolute opacity-0 group-hover:opacity-100 text-white z-10 group-hover:font-semibold group-hover:text-4xl">
```

Comment:
remove group-hover for font-semibold and text-4xl, since they are hidden by opacity